core/silkworm/execution/execution.hpp

```diff
@@ -20,11 +20,11 @@
 #include <vector>
 
 #include <silkworm/chain/config.hpp>
-#include <silkworm/chain/validity.hpp>
 #include <silkworm/execution/processor.hpp>
 #include <silkworm/state/state.hpp>
 #include <silkworm/types/block.hpp>
 #include <silkworm/types/receipt.hpp>
+#include <silkworm/consensus/ethash/ethash.hpp>
```

Comment:
Probably needs consensus_engine.hpp instead of ethash.hpp specifically.

Reply:
Right.